disnake/client.py

```diff
@@ -1974,7 +1974,7 @@ async def fetch_widget(self, guild_id: int, /) -> Widget:
         data = await self.http.get_widget(guild_id)
         return Widget(state=self._connection, data=data)
 
-    async def application_info(self) -> AppInfo:
+    async def application_info(self) -> BotAppInfo:
```

Comment:
Caching this should be up to users, since the data could change at any time - `ext.commands` already caches the owner/owners, which is the only part it currently needs, and the `ready` event only contains app ID + flags :/